src/graphnet/models/detector/prometheus.py

```diff
@@ -26,4 +35,8 @@ def _sensor_pos_z(self, x: torch.tensor) -> torch.tensor:
         return (x + 350) / 100
 
     def _t(self, x: torch.tensor) -> torch.tensor:
-        return ((x / 1.05e04) - 1.0) * 20.0
```

Comment:
Any particular reason why the time normalization function was changed?

Reply:
The previous scaling is sub-optimal for the full dataset. I'll share the full dataset as a CuratedDataset soon.